resources/api-gateway/install.sh

```diff
@@ -62,19 +62,12 @@ else
     --version "$CHART_VERSION" \
     --values values.yaml \
     --namespace apigateway \
-    --create-namespace \
-    --wait
```

Comment:
I had some timeout / reliability issues with this - it works better without waiting